<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArduinoConnect</class>
 <widget class="QWidget" name="ArduinoConnect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>206</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="senderLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>PedalBoard Tasker</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLineEdit" name="SetOffset">
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Set Offset</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="button1_Choice">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>Select Option</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Change Viewer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Slice Offset +</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Slice Offset -</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="button2_Choice">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>Select Option</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Change Viewer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Slice Offset +</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Slice Offset -</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="button3_Choice">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>Select Option</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Change Viewer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Slice Offset +</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Slice Offset -</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="StartButton">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="button1_Label">
     <property name="text">
      <string>Button 1</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="button2_Label">
     <property name="text">
      <string>Button 2</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="button3_Label">
     <property name="text">
      <string>Button 3</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
